src/Rewrite_Command.php

```diff
@@ -218,55 +224,59 @@ public function list_( $args, $assoc_args ) {
-		$rewrite_rules_by_source['date'] = $wp_rewrite->generate_rewrite_rules( $wp_rewrite->get_date_permastruct(), EP_DATE );
-		$rewrite_rules_by_source['root'] = $wp_rewrite->generate_rewrite_rules( $wp_rewrite->root . '/', EP_ROOT );
+		$rewrite_rules_by_source             = array();
+		$rewrite_rules_by_source['post']     = $wp_rewrite->generate_rewrite_rules( $wp_rewrite->permalink_structure, EP_PERMALINK );
+		$rewrite_rules_by_source['date']     = $wp_rewrite->generate_rewrite_rules( $wp_rewrite->get_date_permastruct(), EP_DATE );
+		$rewrite_rules_by_source['root']     = $wp_rewrite->generate_rewrite_rules( $wp_rewrite->root . '/', EP_ROOT );
 		$rewrite_rules_by_source['comments'] = $wp_rewrite->generate_rewrite_rules( $wp_rewrite->root . $wp_rewrite->comments_base, EP_COMMENTS, true, true, true, false );
-		$rewrite_rules_by_source['search'] = $wp_rewrite->generate_rewrite_rules( $wp_rewrite->get_search_permastruct(), EP_SEARCH );
-		$rewrite_rules_by_source['author'] = $wp_rewrite->generate_rewrite_rules($wp_rewrite->get_author_permastruct(), EP_AUTHORS );
-		$rewrite_rules_by_source['page'] = $wp_rewrite->page_rewrite_rules();
+		$rewrite_rules_by_source['search']   = $wp_rewrite->generate_rewrite_rules( $wp_rewrite->get_search_permastruct(), EP_SEARCH );
+		$rewrite_rules_by_source['author']   = $wp_rewrite->generate_rewrite_rules( $wp_rewrite->get_author_permastruct(), EP_AUTHORS );
+		$rewrite_rules_by_source['page']     = $wp_rewrite->page_rewrite_rules();
 
 		// Extra permastructs including tags, categories, etc.
 		foreach ( $wp_rewrite->extra_permastructs as $permastructname => $permastruct ) {
 			if ( is_array( $permastruct ) ) {
-				$rewrite_rules_by_source[$permastructname] = $wp_rewrite->generate_rewrite_rules( $permastruct['struct'], $permastruct['ep_mask'], $permastruct['paged'], $permastruct['feed'], $permastruct['forcomments'], $permastruct['walk_dirs'], $permastruct['endpoints'] );
+				$rewrite_rules_by_source[ $permastructname ] = $wp_rewrite->generate_rewrite_rules( $permastruct['struct'], $permastruct['ep_mask'], $permastruct['paged'], $permastruct['feed'], $permastruct['forcomments'], $permastruct['walk_dirs'], $permastruct['endpoints'] );
 			} else {
-				$rewrite_rules_by_source[$permastructname] = $wp_rewrite->generate_rewrite_rules( $permastruct, EP_NONE );
+				$rewrite_rules_by_source[ $permastructname ] = $wp_rewrite->generate_rewrite_rules( $permastruct, EP_NONE );
 			}
 		}
 
 		// Apply the filters used in core just in case
-		foreach( $rewrite_rules_by_source as $source => $source_rules ) {
-			$rewrite_rules_by_source[$source] = apply_filters( $source . '_rewrite_rules', $source_rules );
-			if ( 'post_tag' == $source )
-				$rewrite_rules_by_source[$source] = apply_filters( 'tag_rewrite_rules', $source_rules );
+		foreach ( $rewrite_rules_by_source as $source => $source_rules ) {
+			// phpcs:ignore WordPress.NamingConventions,PrefixAllGlobals.DynamicHooknameFound
```

Comment:
Would this happen to be a WP Core hook ? If so, please mention that with the whitelist comment.
```suggestion
			// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.DynamicHooknameFound -- Using WP native hook.
```